lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1028,11 +1056,6 @@ func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 			StartingFeeRate: feerate,
 			Budget:          btcutil.Amount(in.Budget),
 		}
-		if in.TargetConf != 0 {
```

Comment:
We still need to set the `params.DeadlineHeight`:

```go
			params.DeadlineHeight = fn.Some(
				int32(in.DeadlineDelta) + currentHeight,
			)
```

Reply:
but this is returned here?

Reply:
correct my bad.